source/templates.tex

```diff
@@ -1390,16 +1388,14 @@
   T& elem(int i) { return v[i]; }
 };
 \end{codeblock}
-
-\pnum
 The prefix
 \tcode{template}
 \tcode{<class}
 \tcode{T>}
 specifies that a template is being declared and that a
 \grammarterm{type-name}
 \tcode{T}
-will be used in the declaration.
+will be used in the definition.
```

Comment:
The "WILL" part seems dubious to me. I'd go for "MAY be used in its declaration", because the sentence does sound like a general-ish description of the meaning of the "template <class T>" prefix to me.